custom/conf/app.ini.sample

```diff
@@ -265,9 +265,9 @@ COOKIE_REMEMBER_NAME = gitea_incredible
 REVERSE_PROXY_AUTHENTICATION_USER = X-WEBAUTH-USER
 ; The minimum password length for new Users
 MIN_PASSWORD_LENGTH = 6
-; True when users are allowed to import local server paths
+; Set to true when users are allowed to import local server paths
```

Comment:
I think this should be: 'Set to true to allow users to import local server paths', because a 'when' construction sounds circumstantial but you can actually decide yourself.

Reply:
Thanks for suggestion. I've updated.